src/main/webapp/scripts/map.js

```diff
@@ -154,31 +163,58 @@ function getColourMarker(category) {
 }
 
 /* Place Marker Where Map is Clicked On & Show Popup*/
-function placeMarkerAndPanTo(latLng, map) {
-  var marker = new google.maps.Marker({
+function placeMarkerAndPanTo(latLng) {
+  /* Make marker for the clicked location */
+  const marker = new google.maps.Marker({
     position: latLng,
     map: map,
     icon: greyIcon
   });
-
+  /* Recenter the Map */
   map.panTo(latLng);
-  togglePopup();
+  /* Show the popup box */
+  togglePopup(latLng);
+  /* Update the latitude and longitude values in the popup */
   var markerPosition = marker.getPosition();
   populateLocation(markerPosition);
```

Comment:
```suggestion
  populateLocation(marker.getPosition());
```

Reply:
This was just a nit. If we don't use a value more than once and calculating the value isn't complicated, usually it's better not to declare a new variable for that.

Reply:
Thanks!